hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson

```diff
@@ -0,0 +1,80 @@
+// Copyright lowRISC contributors.
```

Comment:
Can this file be moved to `hw/top_earlgrey/dv` instead? That will make it match our directory structure scheme.